nodes/src/nodes/llm_gmi/gmi_client.py

```
@@ -0,0 +1,103 @@
+            temperature=0,
+            max_tokens=self._modelOutputTokens,
+        )
+
+        bag['chat'] = self
+
+    def is_retryable_error(self, error):
+        """Determine if the error is retryable."""
+        if isinstance(error, AuthenticationError):
+            return False
+        elif isinstance(error, APIError):
+            return False
```

Ruff (0.15.7)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)